common/core/web/tools/sentry-manager/src/index.ts

```diff
@@ -0,0 +1,137 @@
+///<reference path="../node_modules/@sentry/browser/build/bundle.min.js" />
+
+namespace com.keyman {
+  /**
+   * Controls whether or not the generated Sentry event is logged to the console (true)
+   * or sent to our Sentry server (false).
+   */
+  let DEBUG = false;
+
+  type Options = typeof KeymanSentryManager.DEFAULT_OPTIONS;
+
+  export class KeymanSentryManager {
+    keymanPlatform: string;
+
+    static STANDARD_ALIASABLE_FILES = {
+      'keymanweb.js':    'keymanweb.js',
+      'kmwuibutton.js':  'kmwuibutton.js',
+      'kmwuifloat.js':   'kmwuifloat.js',
+      'kmwuitoggle.js':  'kmwuitoggle.js',
+      'kmwuitoolbar.js': 'kmwuitoolbar.js'
+      // Also add entries for the naming system used by Android and iOS - and map them to the EMBEDDED upload, not the std 'native' one.
```

Comment:
Is this a TODO?

Reply:
Yes; that's a note for what I'll need to do for integration with the apps, which is left to future PRs.